nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -27,6 +27,9 @@
   This also allows configuring runtime settings of AMDVLK and enabling experimental features.
 - The `moonlight-qt` package ([Moonlight game streaming](https://moonlight-stream.org/)) now has HDR support on Linux systems.
 
+- [Sched-ext](https://github.com/sched-ext/scx), a Linux kernel feature to run schedulers in userspace, is now available [services.scx](options.html#opt-services.scx.enable).
```

Comment:
"available _as_ `services.scx`", and code block the option name.

Reply:
You can just do `[](#opt-…)` and it will be filled out automatically, AIUI.

Reply:
> You can just do `[](#opt-…)` and it will be filled out automatically, AIUI.

Even on GitHub?

Reply:
> I'd rather hold this off until kernel 6.12 is stable, at least.

Won't this hard fail on kernels that are not supported though?

I don't think holding this would contribute to its testing/stability. Letting it run on nixos-unstable will attract more bug reports if something is wrong. And once something is merged/approved on upstream kernel, there is little to no change that it will be reverted.